distributor-widget/integrations.md

```diff
@@ -131,6 +131,8 @@ Some events expose additional data layer variables. They are described separatel
 | necessary | boolean | required | Necessary functionality. |
 | performance | boolean | required | Performance, e.g. collecting information about how user uses a website, data are anonymized. |
 
+---
+
```

Comment:
@dobras @dmarchuk so you'd solve it like this? What do you think?

Reply:
The link is ok. This i dont know, we should look at it and unify use of it across all documentation i guess.

Reply:
@zdrazil Yes I think it's better this way. One thing that could be improved in this PR as well is that the section afterwards would have a heading explaining that those are the actual events happening.

Reply:
@dobras otherwise I need to move TrackingConsents section somewhere else. I need somehow separate TrackingConsents from distributorLoaded etc.

@dmarchuk Well the issue with adding a heading for section afterwards is that the structure now is:

### Triggers

#### Data layer variables in events

#### TrackingConsents 

#### distributorLoaded 

etc.

so all this content, even `distributorLoaded` belongs under Triggers section. It can't have a different one.

It's kind of messed up, because triggers basically means events. There's a mix up of terms I think, but the structure currently makes more sense without new heading.

Reply:
What do you think?

Reply:
@dmarchuk mentioned that the whole document structure could be improved and I agree. I'm going to merge this and structure will be improved/discussed later.

Reply:
@zdrazil Created a task for improving the structure of this whole document: https://mews.myjetbrains.com/youtrack/issue/TDD-1829.